### PR TITLE
Add support for running the operator out of cluster 

### PR DESCRIPTION
**Pull request by andersjanmyr**
I like to be able to test locally before I deploy to the cluster. This PR allows this through an environment variable.

```
# Test the app-operator locally by setting KUBERNETES_CONFIG
$ KUBERNETES_CONFIG=~/.kube/config go run cmd/app-operator/main.go
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +47,9 @@ $ cd $GOPATH/src/github.com/example-inc/
 $ operator-sdk new app-operator --api-version=app.example.com/v1alpha1 --kind=App
 $ cd app-operator
 
+# Test the app-operator locally by setting KUBERNETES_CONFIG
+$ KUBERNETES_CONFIG=~/.kube/config go run cmd/app-operator/main.go
```
Comment (fanminshi) [inline README.md]: Let's not add this `KUBERNETES_CONFIG=~/.kube/config go run cmd/app-operator/main.go` to the readme yet.

I want to wrap the above in a new `operator-sdk up` command. I will add that in a separate pr.

**Comment by fanminshi** [comment]
Also follow the [commit style](https://github.com/operator-framework/operator-sdk/blob/master/CONTRIBUTING.MD#format-of-the-commit-message) here for your commits messages.

**Comment by andersjanmyr** [comment]
👍

**Comment by fanminshi** [comment]
@andersjanmyr Could you update the pr with my suggestion? or I can do it for you in this https://github.com/operator-framework/operator-sdk/pull/220.

**Comment by fanminshi** [comment]
lgtm

**Pull request merged**